web_dynamic/0-hbnb.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3
+""" Starts a Flash Web Application """
+from flask import Flask, render_template
+from models import storage
+from models.amenity import Amenity
+from models.city import City
+from models.place import Place
+from models.state import State
+from os import environ
+import uuid
+app = Flask(__name__)
+
+
+@app.teardown_appcontext
+def close_db(error):
+    """ Remove the current SQLAlchemy Session """
+    storage.close()
+
+
+@app.route('/hbnb', strict_slashes=False)
```

Comment:
Hi @oscargomez87,

I'm checking and the route had to be updated to `/0-hbnb/`.

Reply:
route updated